data-otservbr-global/scripts/talkactions/gm/clean.lua

```diff
@@ -1,10 +1,13 @@
 local clean = TalkAction("/clean")
 
 function clean.onSay(player, words, param)
-	if not player:getGroup():getAccess() or player:getAccountType() < ACCOUNT_TYPE_GOD then
+	if not player:getGroup():getAccess() or player:getAccountType() < ACCOUNT_TYPE_GAMEMASTER then
```

Comment:
Probably keep this good only
```suggestion
	if not player:getGroup():getAccess() or player:getAccountType() < ACCOUNT_TYPE_GOD then
```